ctapipe/reco/HillasReconstructor.py

```diff
@@ -99,10 +98,14 @@ class that reconstructs the direction of an atmospheric shower
 
     """
 
-    def __init__(self, config=None, parent=None, **kwargs):
+    def __init__(self, event=None, config=None, parent=None, **kwargs):
```

Comment:
How can event be None?

Reply:
You're right, it shouldn't be a default argument at all, otherwise it cannot work